x/staking/types/delegation.go

```diff
@@ -114,6 +114,11 @@ func (e UnbondingDelegationEntry) IsMature(currentTime time.Time) bool {
 	return !e.CompletionTime.After(currentTime)
 }
 
+// OnHold - is the current entry on hold due to external modules
+func (e UnbondingDelegationEntry) OnHold() bool {
+	return e.UnbondingOnHoldRefCount != 0
```

Comment:
Why not '0<'?

Reply:
I was written before adding the check on decrement. I'll change it.